stack/Pulumi.reference.yaml

```diff
@@ -0,0 +1,11 @@
+encryptionsalt: v1:DunweHhmzCI=:v1:tygBZgKrbXA/zgii:KKS16pfoL346CorH2EwN4I7g+siePA==
+config:
+  datasets:archive_age: "90"
+  datasets:customer_id: C010ys3gt
+  datasets:enable_release: "false"
+  datasets:hail_service_account_full: reference-full-612@hail-295901.iam.gserviceaccount.com
+  datasets:hail_service_account_standard: reference-standard-424@hail-295901.iam.gserviceaccount.com
+  datasets:hail_service_account_test: reference-test-306@hail-295901.iam.gserviceaccount.com
+  gcp:billing_project: reference-762817
+  gcp:project: reference-762817
```

Comment:
Not a big deal because I guess this has been deployed, but could we have reused the `cpg-reference` GCP project - I like the short name :)

Reply:
Oh, I forgot that we had `cpg-reference`, I was sure that we have all that under `cpg-common`!

But I'd prefer to follow the standard pattern of naming our datasets. So if we changed to the `cpg-` prefix here, I'd change it for all other datasets: `cpg-prophecy`, `cpg-acute-care`, etc.

Reply:
(Are you sure we have a project named `cpg-reference` though?)

Reply:
@vladsavelyev I think you're right, there's only a `cpg-reference` bucket in the `cpg-common` project; we don't have a `cpg-reference` project.